configs/h2rbox_v2/README.md

```diff
@@ -12,35 +12,35 @@ With the increasing demand for oriented object detection e.g. in autonomous driv
-| ResNet50 (1024,1024,200) | 89.66 |   1x    |   5.50   |      45.9      |  -  |     2      |              [h2rbox_v2-le90_r50_fpn-6x_hrsc](./h2rbox_v2-le90_r50_fpn-6x_hrsc.py)              |                   [model](https://download.openmmlab.com/mmrotate/v1.0/h2rbox_v2/h2rbox_v2-le90_r50_fpn-6x_hrsc/h2rbox_v2-le90_r50_fpn-6x_hrsc-b3b5e06b.pth) | [log](https://download.openmmlab.com/mmrotate/v1.0/h2rbox_v2/h2rbox_v2-le90_r50_fpn-6x_hrsc/h2rbox_v2-le90_r50_fpn-6x_hrsc-20230312_073744.json)                   |
-| ResNet50 (1024,1024,200) | 89.56 |   1x    |   5.50   |      45.9      |  RR  |     2      |              [h2rbox_v2-le90_r50_fpn_rr-6x_hrsc](./h2rbox_v2-le90_r50_fpn_rr-6x_hrsc.py)              |                   [model](https://download.openmmlab.com/mmrotate/v1.0/h2rbox_v2/h2rbox_v2-le90_r50_fpn_rr-6x_hrsc/h2rbox_v2-le90_r50_fpn_rr-6x_hrsc-ee6e851a.pth) | [log](https://download.openmmlab.com/mmrotate/v1.0/h2rbox_v2/h2rbox_v2-le90_r50_fpn_rr-6x_hrsc/h2rbox_v2-le90_r50_fpn_rr-6x_hrsc-20230312_073800.json)                   |
+|         Backbone         | AP50  | lr schd | Mem (GB) | Inf Time (fps) | Aug | Batch Size |                                   Configs                                   |                                                                     Download                                                                     |
+| :----------------------: | :---: | :-----: | :------: | :------------: | :-: | :--------: | :-------------------------------------------------------------------------: | :----------------------------------------------------------------------------------------------------------------------------------------------: |
+| ResNet50 (1024,1024,200) | 89.66 |   1x    |   5.50   |      45.9      |  -  |     2      |    [h2rbox_v2-le90_r50_fpn-6x_hrsc](./h2rbox_v2-le90_r50_fpn-6x_hrsc.py)    |    [model](https://download.openmmlab.com/mmrotate/v1.0/h2rbox_v2/h2rbox_v2-le90_r50_fpn-6x_hrsc/h2rbox_v2-le90_r50_fpn-6x_hrsc-b3b5e06b.pth)    |
+| ResNet50 (1024,1024,200) | 89.56 |   1x    |   5.50   |      45.9      | RR  |     2      | [h2rbox_v2-le90_r50_fpn_rr-6x_hrsc](./h2rbox_v2-le90_r50_fpn_rr-6x_hrsc.py) | [model](https://download.openmmlab.com/mmrotate/v1.0/h2rbox_v2/h2rbox_v2-le90_r50_fpn_rr-6x_hrsc/h2rbox_v2-le90_r50_fpn_rr-6x_hrsc-ee6e851a.pth) |
 
 ## Citation
 
 ```
 @misc{yu2023h2rboxv2,
-title={H2RBox-v2: Boosting HBox-supervised Oriented Object Detection via Symmetric Learning}, 
+title={H2RBox-v2: Boosting HBox-supervised Oriented Object Detection via Symmetric Learning},
 author={Yi Yu and Xue Yang and Qingyun Li and Yue Zhou and Gefan Zhang and Junchi Yan and Feipeng Da},
```

Comment:
Yi Yu and Xue Yang and Qingyun Li and Yue Zhou and Gefan Zhang and Feipeng Da and Junchi Yan